vmm/src/x64/amd/vs_t.hpp

```diff
@@ -349,7 +349,7 @@ namespace microv
             bsl::expects(mut_sys.bf_vs_op_write(vsid, mk::bf_reg_t_cr3, {}));
             bsl::expects(mut_sys.bf_vs_op_write(vsid, mk::bf_reg_t_cr4, {}));
             bsl::expects(mut_sys.bf_vs_op_write(vsid, mk::bf_reg_t_cr8, {}));
-            bsl::expects(mut_sys.bf_vs_op_write(vsid, mk::bf_reg_t_xcr0, {}));
+            bsl::expects(mut_sys.bf_vs_op_write(vsid, mk::bf_reg_t_cr0, {}));
```

Comment:
Wait what? There should be NO changes to the vs_t.... and this change is VERY wrong. Please revert all changes to the vs_t. There are several and non of these should be here.